extra/test/release_notes_mender-artifact.txt

```diff
@@ -0,0 +1,10 @@
+## mender-artifact 3.6.1
+
+_Released 09.28.2021_
+
+### Statistics
+
+A total of 0 lines added, 0 removed (delta 0)
+
+### Changelogs
+
```

Comment:
These (here but also mender-cli and mender-connect) do not look correct to me. There must be statistics and changelogs.

Reply:
That depends. What you mean is that this file should be removed actually, since there are no changes. This makes sense for releases, where it's a manual process, and we just won't paste it if it says "0 lines".

But [QA-310](https://tracker.mender.io/browse/QA-310) says to "completely replace" what's there, which I take to mean that it shouldn't try to mix it in with the content which is already in mender-docs-changelog, so if nothing is generated, this component won't even exist in the menu. We could try to mix it in with the pre-existing text, but it's more error prone. What do you think?

Reply:
It's good that you pointed it out though, because it looks like they *always* are zero, even when they shouldn't be. I'll fix that, but my point above still stands.

Reply:
Yes, what I meant is that this specific release, `mender-artifact 3.6.1` cannot be blank.

Reply:
No, because mender-artifact in integration 3.0.1 is 3.6.1, and in integration 3.1.0, it's also 3.6.1. So the queried range is `3.6.1..3.6.1`, in other words empty. What I could do is to change the test to test integration `3.0.0..3.1.0` instead of `3.0.1..3.1.0`. This diff would then not be empty anymore, but some of the others still would be, so this might be a test with better coverage of the corner cases. I think I'll do that.

Reply:
Done! See https://github.com/mendersoftware/integration/pull/1836/commits/0f34e8573b3937ae3ee1ed462806368b0f30e122.

Note that mender-cli is still empty, as it should be.

Reply:
I understand it now! Thanks.

Yes, that range seems to be better for test purposes.